src/jsmodules/add.js

```diff
@@ -0,0 +1,20 @@
+import { uPStoreValues } from './storevalues.js';
+import { ul } from './selectors.js';
+
+export class TaskAdd {
+  constructor(task, bool, id) {
+    this.task = task;
+    this.bool = bool;
+    this.id = id;
+  }
+}
+
+export const addItem = (item, array) => {
+  const sizeArr = JSON.parse(localStorage.getItem('taskList')).length;
+  const itemAdd = new TaskAdd(item, true, sizeArr);
```

Comment:
- [ ] Let's take a look at the screenshot below:
![Todos](https://user-images.githubusercontent.com/45002654/144357798-db827e0d-d700-4f76-a263-63f9a427580d.png)

Take a keen look at the `id` property that I've suggested you rename to `index`. Referring back to the requirement quoted in the above review thread, the value of this property should be the value of the **`NEW`** **_array.length_**. Mark the word NEW. **_This means that if I add a task for the first time, my new array will have a length of one, and therefore the index property of my first task will be one._**

In the requirement, we have an example stating that, if you're adding a 5th task to the list, the index of that task should equal to 5. But, the fifth task in the screenshot above has an `id` value of 4.

## Suggestion
In your logic simply add (+ 1) to the length of the array. This means you `refresh` method will need to refactored as suggested in the review thread below:

Reply:
done